standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/CompactionTxnHandler.java

```diff
@@ -1203,6 +1252,79 @@ private static class RetentionCounters {
     }
   }
 
+  static class TxnCleanupQueueHandler {
```

Comment:
should it be renamed to xxUtil or transformed to a singleton?

Reply:
This class is no longer required, since the table is removed.